src/components/RegistrationSuccessForm/styles.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../styles/main.scss";
+
+.container {
+    min-height: 100vh;
+    @include flex-center();
+}
+
+.success-form {
```

Comment:
According to the comment above, what do you think about naming it `sign-up-success`?